src/dialogs/zha-device-zigbee-signature-dialog/dialog-zha-device-zigbee-info.ts

```diff
@@ -0,0 +1,117 @@
+import {
+  LitElement,
+  html,
+  css,
+  CSSResult,
+  TemplateResult,
+  customElement,
+  property,
+  query,
+} from "lit-element";
+import "@polymer/paper-dialog-scrollable/paper-dialog-scrollable";
+import "../../components/dialog/ha-paper-dialog";
+import "../../components/ha-code-editor";
+// Not duplicate, is for typing
+// tslint:disable-next-line
+import { HaPaperDialog } from "../../components/dialog/ha-paper-dialog";
+
+import { PolymerChangedEvent } from "../../polymer-types";
+import { haStyleDialog } from "../../resources/styles";
+import { HomeAssistant } from "../../types";
+import { ZHADeviceZigbeeInfoDialogParams } from "./show-dialog-zha-device-zigbee-info";
+// tslint:disable-next-line: no-duplicate-imports
+import { HaCodeEditor } from "../../components/ha-code-editor";
+import { afterNextRender } from "../../common/util/render-status";
+import { fireEvent } from "../../common/dom/fire_event";
+
+@customElement("dialog-zha-device-zigbee-info")
+class DialogZHADeviceZigbeeInfo extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() private _params?: ZHADeviceZigbeeInfoDialogParams;
+  @property() private _error?: string;
+  @property() private _signature: any;
+  @query("ha-code-editor") private _codeEditor?: HaCodeEditor;
+  @query("ha-paper-dialog") private _dialog!: HaPaperDialog;
+
+  public async showDialog(
+    params: ZHADeviceZigbeeInfoDialogParams
+  ): Promise<void> {
+    this._params = params;
+    const signature = JSON.parse(JSON.stringify(this._params.device.signature));
+    signature.manufacturer = this._params.device.manufacturer;
+    signature.model = this._params.device.model;
+    this._signature = JSON.stringify(signature, undefined, 2);
+    await this.updateComplete;
+    this._dialog.open();
+    afterNextRender(() => {
+      if (this._codeEditor?.codemirror) {
+        this._codeEditor.codemirror.refresh();
+      }
+      afterNextRender(() =>
+        fireEvent(this._dialog as HTMLElement, "iron-resize")
+      );
+    });
+  }
+
+  protected render(): TemplateResult {
+    if (!this._params) {
+      return html``;
+    }
+
+    return html`
+      <ha-paper-dialog
```

Comment:
We should stop adding new `ha-paper-dialog`s and only add `ha-dialog`s (mwc)

Reply:
Will not block this now.

Reply:
I can do it in this. I don't mind. I'll look into it.

Reply:
Think i got it...